src/systems/ec/implementations/react/templates/pages/person-list/src/PersonListPage.jsx

```diff
@@ -0,0 +1,527 @@
+                      natasha.bertaud@ec.europa.eu
+                    </a>
+                  </DescriptionDefinition>
+                  <DescriptionTerm>Social media</DescriptionTerm>
+                  <DescriptionDefinition className="ecl-u-mv-none ecl-u-pv-none">
+                    <SocialMediaFollow
+                      className="ecl-u-mt-m ecl-u-pa-none ecl-u-bg-white"
+                      links={[
+                        {
+                          href: '/example',
+                          label: 'Twitter',
+                          variant: 'standalone',
+                          iconPosition: 'before',
+                          icon: [
+                            {
+                              shape: 'twitter',
+                              size: 's',
+                            },
+                            {
+                              shape: 'twitter_hover',
+                              size: 's',
+                            },
+                          ],
+                        },
+                        {
+                          href: '/example',
+                          label: 'Facebook',
+                          variant: 'standalone',
+                          iconPosition: 'before',
+                          icon: [
+                            {
+                              shape: 'facebook',
+                              size: 's',
+                            },
+                            {
+                              shape: 'facebook_hover',
+                              size: 's',
+                            },
+                          ],
+                        },
+                      ]}
+                    />
+                  </DescriptionDefinition>
+                  <DescriptionTerm>Office</DescriptionTerm>
+                  <DescriptionDefinition>BERL 04/353</DescriptionDefinition>
+                  <DescriptionTerm>Spokesperson</DescriptionTerm>
+                  <DescriptionDefinition>
+                    <UnorderedList>
+                      <UnorderedListItem>
+                        activities of the president
+                      </UnorderedListItem>
+                      <UnorderedListItem>
+                        policies of the first vice-president
+                      </UnorderedListItem>
+                      <UnorderedListItem>
+                        better regulation, interinstitutional affairs
+                      </UnorderedListItem>
+                      <UnorderedListItem>
+                        migration, home affairs and citizenship
+                      </UnorderedListItem>
+                      <UnorderedListItem>security union</UnorderedListItem>
+                    </UnorderedList>
+                  </DescriptionDefinition>
+                </DescriptionList>
+              </div>
+            </article>
+
+            <SocialMediaShare
+              className="ecl-u-mt-none ecl-u-mb-xl ecl-u-mb-lg-3xl"
```

Comment:
Same, `ecl-u-mb-xl ecl-u-mb-lg-3xl` Should be removed as we have already bottom margin from the main content container